docs/src/content/sending-messages.mdx

```diff
@@ -34,9 +34,56 @@ gets sent out. You might use personalisation to:
 - Address a user by their name
 - Provide users with a specific link to click
 - Send a unique transaction number as a follow up
+- Give users a dynamically generated list of actions they need to take
 
 Your API call must include any fields that have been set up as personalisations.
 
+### List fields
+
+A list field allows you to include a nicely formatted list of things in your
+message. A list field looks just like an ordinary field in your template.
+
+For example, your Notify template might look like this:
+
+```text
+Dear ((name)), please bring these documents to your appointment:
+
+((documents))
+```
+
+In this example, `((documents))` looks just like `((name))` but its intent is to
+become a nicely formatted list of documents.
+
+Notify allows you to convert this field into a list in the generated message.
+To do this, your code should pass in a list of things as the value of the
+personalisation field, instead of a string.
+
+For example, in Go you would pass a slice of strings:
+
+```go
+p := notify.Personalisation{
+	{"name", "Kim"},
+	{"documents", []string{"Driver licence", "Medicare card"}},
+}
+```
+
+Generated emails will display the list as bullet points:
+
+```text
+Dear Kim, please bring these documents to your appointment:
+
+• Driver licence
+• Medicare card
+```
+
+Generated text messages will display the list inline:
+
+```text
+Dear Kim, please bring these documents to your appointment:
+
+Driver licence and Medicare card
```

Comment:
If there were more than 2 items in the list, would it become `Driver license, xxx1, xxx2 and Medicare card`?

Reply:
Yeah. Should my example use 3 items?